app/controllers/concerns/foreman_rh_cloud/iop_smart_proxy_access.rb

```diff
@@ -0,0 +1,19 @@
+module ForemanRhCloud
+  module IopSmartProxyAccess
+    extend ActiveSupport::Concern
+
+    private
+
+    def require_no_iop_smart_proxy
```

Comment:
I think "non_iop" is more clear
```suggestion
    def require_non_iop_smart_proxy
```

Reply:
How about _iopless_ :)